### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by Sauder91** [AndreiMisiukevich/CardView#188]
I've noticed some Accessibility issues and inconsistencies.  

The big one is it would be great if the CardsViews/CarouselViews was selectable by  responded to the appropriate accessibility interaction (adjustable on iOS so you could swipe up and down to navigate the view with accessibility, https://developer.apple.com/documentation/uikit/accessibility/delivering_an_exceptional_accessibility_experience)(TYPE_VIEW_SCROLLED on Android so the change page gesture would change the CardView Page https://developer.android.com/guide/topics/ui/accessibility/custom-views.html).  Along with this right now if you make the whole CardView Available for accessibility on iOS you can't make anything under it accessible.  

it would be great if the changing the CardView Page either read the first element with a Accessibility Labels label or you could somehow set text to be read for each page evertime its diplayed.  Take away, you should be able to have accessibility read the text you want when a page a switched to.  This one would be nice but I could probably accomplish this myself by listening for page changes.

It would also be nice if there was a property you could set so that all of a CardView pages childrens accessibility text was read back all at once.  Android seems to do a good job of this by default but on iOS you have to go through each item.  Again on this one,  it would be nice if the nuget just did it but I could probably accomplish it myself.

A few small things,  accessibility text on the arrows to change page isn't the best, and by default you can access them still after they fade.  If you make the page indicators available for accessibility you can only access them as a group not individually.

**Comment by AndreiMisiukevich** [comment]
Hey, thanks for reaction)
I will reply on weekend

**Comment by AndreiMisiukevich** [comment]
sorry, it's crossplatform controls (iOS, Android, UWP, Tizen, GTK, Mac). I am not keen to support all platform-specific features for now
